hive-llap/llap.sh

```diff
@@ -34,30 +34,32 @@ readonly HIVE_CONF_DIR='/etc/hive/conf'
 readonly REGION=$(/usr/share/google/get_metadata_value attributes/dataproc-region)
 readonly DEFAULT_INIT_ACTIONS_REPO="gs://dataproc-initialization-actions"
 
-##user supplied location for file to ingest
+# user supplied location for file to ingest
 readonly INIT_ACTIONS_REPO="$(/usr/share/google/get_metadata_value attributes/init-actions-repo || echo ${DEFAULT_INIT_ACTIONS_REPO})"
-##directory files ingestied will reside
-readonly INIT_ACTIONS_DIR=$(mktemp -d -t dataproc-init-actions-XXXX)
+# directory files ingestied will reside
+readonly INIT_ACTIONS_DIR='/usr/lib/dataproc-init-actions/hive-llap'
```

Comment:
We don't need to have init actions directory. Can be something like `/usr/lib/hive-llap`

Reply:
changed.